src/main/webapp/scripts/cookies.js

```diff
@@ -0,0 +1,85 @@
+const HOURS_PER_DAY = 24;
+const MINUTES_PER_HOUR = 60;
+const SECONDS_PER_MINUTE = 60;
+const SECONDS_PER_DAY = HOURS_PER_DAY*MINUTES_PER_HOUR*SECONDS_PER_MINUTE;
```

Comment:
[Nit] Put spaces on each side of a binary operator
e.g. `x = 2 + 2` instead of `x=2+2`